src/plugins/intel_gpu/src/graph/impls/onednn/fully_connected_onednn.cpp

```diff
@@ -69,11 +68,9 @@ struct fully_connected_onednn : typed_primitive_onednn_impl<fully_connected> {
                 args.insert({DNNL_ARG_ATTR_SCALES | DNNL_ARG_WEIGHTS, scale_mem->get_onednn_memory(desc)});
             }
 
```

Comment:
it would be better to add an assert not to accept zero_point_scalar

Reply:
zero_point_scalar should be set because it couldn't know which impl will be selected when creating fc primitive.